### PR TITLE
Check only org members

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

usersmapchecker can check if author of PR is a organisation member for which PR is created.

### CODE REVIEW

File: development/tools/cmd/usersmapchecker/main.go [hunk 1]
```diff
@@ -97,30 +103,46 @@ func main() {
...
 	for _, author := range authors {
-		// Use goroutines.
-		go func(wg *sync.WaitGroup, author string, exitCode *atomic.Value) {
-			// Notify goroutine is done when exiting from it.
-			defer wg.Done()
-			for _, user := range usersMap {
-				if user.ComGithubUsername == author {
-					contextLogger.LogInfo(fmt.Sprintf("user %s is present in users map", author))
-					return
+		member, _, err := githubComClient.Organizations.IsMember(ctx, org, author)
```
Comment (Halamix2) [inline development/tools/cmd/usersmapchecker/main.go:119]: if `err` is unused it can be removed here

Comment (dekiel) [reply development/tools/cmd/usersmapchecker/main.go:119]: :) It should be used. Will add some body to the if block.

Comment (dekiel) [reply development/tools/cmd/usersmapchecker/main.go:119]: @Halamix2 Added missing error handling block.

**Pull request merged**